### ISSUE TITLE
Miss you

### ISSUE THREAD

**Issue by modaresimr** [agata/dl.biscuit#48]
We are really missing you.
This application is really useful and good.
If you do not have time to continue this great project would you please share the source code? so we can improve it :)

**Comment by niutech** [comment]
Technically, the source code is in `%LOCALAPPDATA%\Programs\biscuit\resources\app.asar`. You can unpack it using https://github.com/electron/asar. Or open Biscuit and click `Ctrl-Shift-I` for Dev Tools. But it is not open source.

**Comment by modaresimr** [comment]
You are right, but it is not as clear to understand and develop as the original source code and it is hard to improve it.
